caluma/caluma_analytics/schema.py

```diff
@@ -84,6 +84,15 @@ def resolve_records(table, info, *args, **kwargs):
     serializers.StartingObjectField, lambda field: StartingObject
 )
 
+TableType = graphene.Enum(
+    "TableType",
+    [(key.upper(), key) for key, _ in models.AnalyticsTable.TABLE_TYPE_CHOICES],
+)
+
+serializer_converter.get_graphene_type_from_serializer_field.register(
+    serializers.TableTypeField, lambda field: TableType
+)
```

Comment:
This is a pattern I've seen alot of times in our codebase, is there maybe a way to abstract this?

Reply:
Absolutely!